### ISSUE TITLE
`tslint:disable-next-line` disables two subsequent lines

### ISSUE THREAD

**Issue by dylanpyle** [palantir/tslint#3174]
### Bug Report

- __TSLint version__:  5.7.0
- __TypeScript version__: 2.4.2
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
/* tslint:disable-next-line:max-line-length */
const a = '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111';
const b = '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111';
const c = '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111';
const d = '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111';
const e = '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111';
const f = '11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111';
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "max-line-length": [true, 50]
  }
}
```

#### Actual behavior
```
ERROR: foo.ts[4, 1]: Exceeds maximum line length of 50
ERROR: foo.ts[5, 1]: Exceeds maximum line length of 50
ERROR: foo.ts[6, 1]: Exceeds maximum line length of 50
ERROR: foo.ts[7, 1]: Exceeds maximum line length of 50
```

#### Expected behavior
```
ERROR: foo.ts[3, 1]: Exceeds maximum line length of 50
ERROR: foo.ts[4, 1]: Exceeds maximum line length of 50
ERROR: foo.ts[5, 1]: Exceeds maximum line length of 50
ERROR: foo.ts[6, 1]: Exceeds maximum line length of 50
ERROR: foo.ts[7, 1]: Exceeds maximum line length of 50
```

No matter where I place the `disable-next-line` rule in the file, it always seems to affect the *two* subsequent lines, not just the line immediately afterwards. I don't see this behavior in the docs or referenced in any other issues — sorry if I missed something!

**Comment by ajafff** [comment]
I can confirm that this is a bug.
What's happening here: the disabled range goes up to the start of the next line. If the failure begins at the first character in that line, the error is erroneously disabled.

**Comment by dylanpyle** [comment]
@ajafff Thanks for the quick fix! 🎉

**Comment by paulTchaa8** [comment]
Please help dears. When launching the `ng new my-project` , i got this. What could be the issue and how to solve that? I'm learning angular
![issue](https://user-images.githubusercontent.com/40181181/60740485-cf49b580-9f5d-11e9-9a42-6a408530c60f.JPG)
I've tried to install some other asked dependencies, with no effects..

**Comment by JoshuaKGoldberg** [comment]
@paulTchaa8 you'll want to file an issue on [Angular on GitHub](https://github.com/angular/angular). That doesn't look like a TSLint issue.
